packages/server/modules/automate/repositories/automations.ts

```diff
@@ -93,6 +93,28 @@ export async function getFullAutomationRevisionMetadata(
   }
 }
 
+export type InsertableAutomationFunctionRun = Omit<
+  AutomationFunctionRunRecord,
+  'id' | 'runId'
```

Comment:
The id needs to be present in the insert model, otherwise the sql command cannot find the "conflict"